modules/sharedaddy.php

```diff
@@ -20,4 +20,13 @@
 
 function sharedaddy_loaded() {
 	Jetpack::enable_module_configurable( __FILE__ );
+	add_filter( 'jetpack_module_configuration_url_sharedaddy', 'jetpack_sharedaddy_configuration_url' );
+}
+
+function jetpack_sharedaddy_configuration_url() {
+	// get URL from WP
+	$site_url = parse_url( get_site_url() );
+	// remove www. if present
+	$parsed_url = str_replace( 'www.', '', $site_url[ 'host' ] );
+	return 'https://wordpress.com/sharing/buttons/' . $parsed_url;
```

Comment:
Left a comment in slack, but we need to use `Jetpack::build_raw_urls( home_url() ); here since the Jetpack site could be in a folder and we figure out the Calypso link via the home URL.

An alternative would be suffixing the Calypso URL with the Jetpack site ID available via `Jetpack_Options::get_option( 'id' )`

Let's fix this up first thing and slide it in just before we declare the 7.3 feature freeze.

Reply:
Resolved via 0128590